src/containers/layouts/messaging/commentsList1/commentsList1.component.tsx

```diff
@@ -62,7 +62,6 @@ class CommentsList1Component extends React.Component<CommentsList1Props> {
 export const CommentsList1 = withStyles(CommentsList1Component, (theme: ThemeType) => ({
   container: {
     flex: 1,
-    paddingHorizontal: 16,
```

Comment:
now you understand 😄 
I also recommend not using any of flex/margin/padding styles in common components